magefile.go

```diff
@@ -14,6 +14,11 @@ import (
 	"github.com/magefile/mage/sh"
 )
 
+// Runs all the golang tests while checking for race conditions
```

Comment:
Offered as a useful task perhaps should be packaged in a real pull request adding common golang project mage tasks.